tests/sensors/test_time_sensor.py

```diff
@@ -73,3 +73,11 @@ def test_target_time_aware(self):
             op = TimeSensorAsync(task_id="test", target_time=aware_time)
             assert hasattr(op.target_datetime.tzinfo, "offset")
             assert op.target_datetime.tzinfo.offset == 0
+
+    def test_target_time_naive_dag_timezone(self):
+        with DAG(
+            "test_target_time_naive_dag_timezone",
+            start_date=pendulum.datetime(2020, 1, 1, 0, 0, tz=DEFAULT_TIMEZONE),
+        ):
+            op = TimeSensorAsync(task_id="test", target_time=pendulum.time(8, 0))
+            assert op.target_datetime.time() == pendulum.time(0, 0)
```

Comment:
I feel this should also test the target datetime always has a timezone.

Reply:
And done.